### PR TITLE
Create ui-c8y-1020-27-1-information-about-mqtt-rest-limitation-for-shared-trusted-c-6992-graft.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60465](https://cumulocity.atlassian.net/browse/MTM-60465)
Original PR: [7060](https://github.softwareag.com/IOTA/cumulocity-ui/pull/7060)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1020-27-1-information-about-mqtt-rest-limitation-for-shared-trusted-c-6992-graft.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: information about MQTT/REST limitation for shared trusted c… (#6992) [GRAFT][release/cd] (#7060)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-27-1-information-about-mqtt-rest-limitation-for-shared-trusted-c-6992-graft.md]: The Claude AI suggested to use the following title:
```suggestion
title: Information about MQTT/REST limitation for shared trusted certificates
```

File: content/change-logs/platform-services/ui-c8y-1020-27-1-information-about-mqtt-rest-limitation-for-shared-trusted-c-6992-graft.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60465
+version: 1020.27.1
+---
+information about MQTT/REST limitation for shared trusted c… (#6992) [GRAFT][release/cd] (#7060)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-27-1-information-about-mqtt-rest-limitation-for-shared-trusted-c-6992-graft.md]: The Claude AI suggested to use the following body:
```suggestion
In the context of using shared trusted certificates for MQTT and REST connections, there was previously a lack of clear documentation about the limitations and requirements. With this change, the documentation now explicitly states that the use of shared trusted certificates for MQTT and REST connections is not supported. This impacts users by clarifying that they cannot use shared trusted certificates for these connection types, and they will need to use dedicated certificates instead.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@mmed-c8y Please review and modify as needed.

**Comment by BeateRixen** [comment]
> @mmed-c8y Please review and modify as needed.

It seems that the ticket owner has changed to @kgol-c8y :-).

**Comment by kgol-c8y** [comment]
I think we will not need this PR, its already created from Pankaj. I will cross check and close acccordingly

File: content/change-logs/platform-services/ui-c8y-1020-27-1-information-about-mqtt-rest-limitation-for-shared-trusted-c-6992-graft.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60465
+version: 1020.27.1
+---
+The message "Shared trusted certificates are enabled on this instance. Devices may not able to connect over MQTT/REST using certificates." on trusted certificate page appears when "ssl shared truststore" is enabled.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-27-1-information-about-mqtt-rest-limitation-for-shared-trusted-c-6992-graft.md:17]: On the **Trusted certificates** page, an alert has been added which appears if "SSL shared truststore" is enabled: "Shared trusted certificates are enabled on this instance. Devices may not able to connect over MQTT/REST using certificates."

File: content/change-logs/platform-services/ui-c8y-1020-27-1-information-about-mqtt-rest-limitation-for-shared-trusted-c-6992-graft.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: An alert has been added indicating that the connection via MQTT REST may not be possible when using shared trusted certificates.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-27-1-information-about-mqtt-rest-limitation-for-shared-trusted-c-6992-graft.md:3]:
```suggestion
title: Added alert for shared trusted certificates
```

**Comment by BeateRixen** [comment]
@kgol-c8y What is the status here? Do we still need this PR?

**Comment by kgol-c8y** [comment]
@BeateRixen  this PR is not needed, the release notes for the epic is covered in https://github.com/SoftwareAG/c8y-docs/pull/2408/files

**Pull request closed without merging**